runtime/doc/insert.txt

```diff
@@ -1421,9 +1421,9 @@ automatically switch to HTML/CSS/JavaScript completion. Note: contrary to
 original HTML files completion of tags (and only tags) isn't context aware.
 
 
-RUBY							*ft-ruby-omni*
+RUBY						 {Nvim} *ft-ruby-omni*
```

Comment:
If I switch their position the highlighting works, so I think I'll go with that (although it's inconsistent with where the `{Nvim}` tag usually is).